src/core/openthread-core-default-config.h

```diff
@@ -66,6 +66,16 @@
 #endif  // OPENTHREAD_CONFIG_DEFAULT_CHANNEL
 
 /**
+ * @def OPENTHREAD_CONFIG_DEFAULT_MAX_TRANSMIT_POWER
+ *
+ * The default IEEE 802.15.4 maximum transmit power (dBm)
+ *
+ */
+#ifndef OPENTHREAD_CONFIG_DEFAULT_MAX_TRANSMIT_POWER
+#define OPENTHREAD_CONFIG_DEFAULT_MAX_TRANSMIT_POWER        127
```

Comment:
Well, 127dBm is something like 5 gigawatts. haha. More importantly, I believe we are using 127 as a special value that indicates that the power is undefined or otherwise not set. 

Honestly, I think 0 dBm is a reasonable default value. Making the default be the maximum transmit power the device is capable of doesn't seem like a good idea.